resources/assets/less/bem/beatmap-discussions-header-top.less

```diff
@@ -56,6 +56,30 @@
     }
   }
 
+  &__filter-group {
+    min-width: 0;
+    flex: 1 1 auto;
```

Comment:
is the full width for beatmap dropdown intentional?

![](https://s.myconan.net/2018-05/firefox_2018-05-07_17-00-11.png)

Reply:
yes